templates/web/pages/export_products/export_products.tt.js

```diff
@@ -0,0 +1,72 @@
+var poll_n1 = 0;
+var timeout1 = 5000;
+var job_info_state1;
+
+var poll_n2 = 0;
+var timeout2 = 5000;
+var job_info_state2;
+
+var poll_n3 = 0;
+var timeout3 = 5000;
+var job_info_state3;
+
+var minion_status = {
+	"inactive" : "$Lang{minion_status_inactive}{$lc}",
```

Comment:
Inside the templates, there isn't a direct access to variables like $Lang{minion_status_inactive}{$lc} but we can access a few functions available to templates: [% lang{minion_status_inactive") %]